### ISSUE TITLE
Widget - live view of detection

### ISSUE THREAD

**Issue by He3556** [SecUpwN/Android-IMSI-Catcher-Detector#278]
![widget](https://cloud.githubusercontent.com/assets/2507905/5709058/b80d204e-9a96-11e4-9567-57ec2f35fe6e.jpg)

This widget shall be part of our redesign of the UI/UX in #258.

1. Button to open the APP 
2. Button to show less/more of the Widget (if less, only the yellow part will be shown)
3. Up- & Down (for the Live Logview) + optional a stop and clear button
4. Indicator LEDs, green blinks every time the detection runs this routine is. 
      Red is for a positive detection. 
5. Counts each detections. The idea behind – a better qualification of the events:
      * if changed LAC counts more than 1 – it is even worse
      * if unknown Cell counts more than 1 or 2 – OCID has not discovered this area (=no catcher) 
      * if NC List of more than 1 or 2 BTS is empty – there might me another problem (=no catcher)

We can split the development in 2 steps. 
1. Yellow part and the live logview
2. Indicator LEDs  and the rest of the row.

**Comment by SecUpwN** [comment]
Awesome, this would be my new main widget! Quick suggestion for `1`: Use a round [material Icon](https://google.github.io/material-design-icons/).

**Comment by E3V3A** [comment]
Yes, I'd prefer the `1` button to be something between the "settings power" and the "settings backup" buttons on that page. With the following functions as toggled when hitting it:
 1. Red "Power" (off) icon, when app is not running any active scans and log events are off
 2. Green "Power" (on) icon when AIMSICD is actively scanning/running and logging events.

**Comment by He3556** [comment]
yes about the icons @SecUpwN 
and yes @E3V3A the Button no.2 should have another function. Turn on/off is a good idea. 
We could even talk about a On/Off Switch for "Monitoring" and "Tracking"?

**Comment by SecUpwN** [comment]
>We could even talk about a On/Off Switch for "Monitoring" and "Tracking"?

Press once: Turn on/off monitoring, press twice: Turn on/off tracking? Or is that too complex?

**Comment by Davidmylove** [comment]
My ex lady erica casimiro is using your code or what ever to brake in try
stuff . Am contacting a lower monday

**Comment by SecUpwN** [comment]
>My ex lady erica casimiro is using your code or what ever to brake in try stuff . Am contacting a lower monday

@Davidmylove, you've commented on multiple Issues around our GitHub repository now and I simply cannot make any sense from what you are saying here. Are you assuming that your ex boyfriend is using our App to "hack into your stuff"? You've completely misunderstood our App, please refrain from commenting on any further Issues and take care of your broken heart elsewehere. Otherwise I will personally report you. Our App neither hacks any accounts, nor are we suable (see our [DISCLAIMER](https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/blob/master/DISCLAIMER)).

**Comment by Davidmylove** [comment]
@davidmylove no sorry I send this to erica on accident but I rewrote and
sent it to github

**Comment by SecUpwN** [comment]
![Circular Progress Bar](http://www.michenux.net/wp-content/gallery/android/dynamic/circular-progress-bar3-nggid0224-ngg0dyn-300x0x100-00f0w010c010r110f110r010t010.png) ![Circular Progress Bar](http://www.michenux.net/wp-content/gallery/android/dynamic/circular-progress-bar4-nggid0225-ngg0dyn-300x0x100-00f0w010c010r110f110r010t010.png)

The above is something @E3V3A just sent me: [Refresh item inside action bar using circular ProgressBar](http://www.michenux.net/android-refresh-item-action-bar-circular-progressbar-578.html). He thinks that looks pretty neat, maybe we can display this while scanning takes place?

**Comment by E3V3A** [comment]
I want to add that it was just an idea. I'd be quite satisfied with just a blinking dot a well, as long as something is moving to show the user that the app is actually not frozen...

**Comment by SecUpwN** [comment]
>I want to add that it was just an idea.

Here is **another idea** called [CircleProgress](https://github.com/lzyzsd/CircleProgress):

![CircleProgress](https://raw.githubusercontent.com/lzyzsd/CircleProgress/master/demos/circle_progress.gif)

**Comment by SecUpwN** [comment]
@He3556, please stop closing Issues you may think csnnpt be completed yet or do casre developers away. See other big projects and you will notice: Even though the Issue list is huge, they develop stuff and users are able to solve anything open. I still consider this an important feature, I am re-opening this.
